packages/common/tests/customChains.spec.ts

```diff
@@ -90,6 +90,20 @@ tape('[Common]: Custom chains', function (t: tape.Test) {
       )
     }
 
+    common = Common.custom(CustomChain.PolygonMumbai, { hardfork: Hardfork.Byzantium })
```

Comment:
Super nit but can you add a sanity check? I.e. (For the other one too). This ensures the integrity of these tests.

```
```suggestion
    common = Common.custom(CustomChain.PolygonMumbai)
    st.notEqual(common.hardfork(), Hardfork.Byzantium, 'by default PolygonMumbai hardfork is not Byzantium')
```